MotionMark/resources/debug-runner/debug-runner.js

```diff
@@ -528,18 +528,20 @@ window.suitesManager = {
     }
 }
 
-Utilities.extendObject(window.benchmarkController, {
-    initialize: async function()
+class DebugBenchmarkController extends BenchmarkController {
+    async initialize()
```

Comment:
Should not this function calls `super.initialize()`? They both set the version string, add the orientation change listener and detect the frame rate.